certificate/models.py

```diff
@@ -163,6 +163,9 @@ class Esim_faculty(models.Model):
 class Osdag_WS(models.Model):
     name = models.CharField(max_length=200)
     email = models.EmailField()
+    college = models.CharField(max_length=200, null=True, blank=True)
+    ws_date = models.CharField(max_length = 100, null=True, blank=True)
+    ws_year = models.CharField(max_length = 100, null=True, blank=True)
```

Comment:
Keep this ```date``` anad ```year``` only, that's better.
No need of ```ws_```
So ```Osdag_WS.date```  is better than ```Osdag_WS.ws_date```
Even the class name is not good but for now let it be.

Reply:
Done